editoast/src/views/rolling_stocks.rs

```diff
@@ -162,6 +163,15 @@ struct RollingStockLiveryCreateForm {
     pub images: Vec<TempFile>,
 }
 
+#[delete("/{rolling_stock_id}")]
+async fn delete(db_pool: Data<DbPool>, path: Path<i64>) -> Result<HttpResponse> {
+    let rolling_stock_id = path.into_inner();
+    if !RollingStockModel::delete(db_pool, rolling_stock_id).await? {
+        return Err(RollingStockError::NotFound { rolling_stock_id }.into());
+    }
+    Ok(HttpResponse::build(StatusCode::NO_CONTENT).body(""))
```

Comment:
```suggestion
    Ok(HttpResponse::NoContent())
```

Reply:
Since it should return and HttpResponse and not an HttpResponseBuilder, I'll change the code to:
Ok(HttpResponse::NoContent().finish())

Reply:
Oh, you're right. You can also change the return type to `Result<impl Responder>` and actix will call finish() where needed

Reply:
Good to know ! For now, I'd rather have the `finish()` which explicitly indicates that the Err's body is empty.